patches/add-feature-flag-based-on-homeserver/matrix-react-sdk+3.71.1.patch

```diff
@@ -0,0 +1,42 @@
+diff --git a/node_modules/matrix-react-sdk/src/IConfigOptions.ts b/node_modules/matrix-react-sdk/src/IConfigOptions.ts
+index b2e44f2..28ff388 100644
+--- a/node_modules/matrix-react-sdk/src/IConfigOptions.ts
++++ b/node_modules/matrix-react-sdk/src/IConfigOptions.ts
+@@ -192,3 +192,12 @@ export interface ISsoRedirectOptions {
+     immediate?: boolean;
+     on_welcome_page?: boolean;
+ }
++
++
++//:tchap: add tchap specific fields
++export interface IConfigOptions {
++    tchap?: {
++        feature_email_notification:[string]//activate email notification on a list of home servers, ie : "agent1.tchap.incubateur.net"
++    }
++}
++//end :tchap: 
+\ No newline at end of file
+diff --git a/node_modules/matrix-react-sdk/src/components/views/settings/Notifications.tsx b/node_modules/matrix-react-sdk/src/components/views/settings/Notifications.tsx
+index 3e833b3..9010378 100644
+--- a/node_modules/matrix-react-sdk/src/components/views/settings/Notifications.tsx
++++ b/node_modules/matrix-react-sdk/src/components/views/settings/Notifications.tsx
+@@ -48,6 +48,7 @@ import {
+     updatePushRuleActions,
+ } from "../../../utils/pushRules/updatePushRuleActions";
+ import { Caption } from "../typography/Caption";
++import TchapUIFeature from "../../../../../../src/tchap/util/TchapUIFeature";
+ 
+ // TODO: this "view" component still has far too much application logic in it,
+ // which should be factored out to other files.
+@@ -703,8 +704,9 @@ export default class Notifications extends React.PureComponent<IProps, IState> {
+                         />
+                     </>
+                 )}
+-
+-                {emailSwitches}
```

Comment:
would be nice to comment out this code with TCHAP comments, to avoid seeing it reappear accidentally later